public/js/app/adapters/FormDataAdapter.js

```diff
@@ -27,13 +29,31 @@ define(["app/app",
 
       var headers = this.get('headers')
       if (headers !== undefined) {
-        hash.beforeSend = function (xhr) {
+        hash.beforeSend = function(xhr) {
           Ember.keys(headers).forEach(function(key) {
             xhr.setRequestHeader(key, headers[key])
           })
         }
       }
 
+      hash.xhr = function() {
+        var xhr = new window.XMLHttpRequest()
```

Comment:
I wonder if modern jquery has XMLHttpRequest wrapper with fallbacks for browsers that do not support it natively?